app/helpers/application_helper.rb

```diff
@@ -74,11 +74,24 @@ def records_sorted_by_short_name(records)
     end
   end
 
-  def campaign_select_list
+  def campaign_select_list(form:)
     select_list = [ANY_CAMPAIGN_KEYWORD]
 
-    camps = RecordData.select(:value).where(column_name: "Campaigns/Campaign/ShortName").where.not(value: "").order(:value).distinct
-    select_list.concat(camps.map(&:value))
+    if current_user.daac_curator?
+      daac = current_user.daac
+      state = form == home_path ? :in_daac_review : [:finished, :closed]
+    elsif current_user.mdq_curator?
+      daac = MDQ_PROVIDERS
+      state = form == home_path ? [:open, :in_arc_review, :ready_for_daac_review] : [:finished, :closed]
```

Comment:
not anything to be addressed now, but wondering if there should be a `:in_mdq_review` state?

Reply:
I was thinking about that too, but it might just want to be :in_review.  or :in_curator_view

Reply:
Have we written any tickets based on what MDQ curators might want yet?